package.json

```diff
@@ -17,6 +17,8 @@
   "author": "André Restivo, Hugo Sereno Ferreira, Rui Gonçalves",
   "license": "MIT",
   "dependencies": {
+    "rand-seed": "^1.0.0",
+    "set-blocking": "^2.0.0",
```

Comment:
So we need these in `dependencies`? If they are now used only for testing, can't we move them to `devDependencies`

Reply:
No, I forgot to remove them because I didn't use them after all, thank you for pointing it out